### PR TITLE
update(tooltip): Make inverted text readable in dark mode

### PR DESCRIPTION
**Pull request by Xapphire13**
to: @williaster @hayes @alecklandgraf

## Description
When in dark mode, the inverted tooltip text isn't readable. This change fixes that, and also improves the accessibility of the colors for light mode.

In order to make this fix, ~I needed to have theme mode aware styles or use some other color from the theme; there were no applicable analogs, so I went the route of exposing the current theme mode in the theme.~ I added a new `baseInverse` color value to the theme.

## Motivation and Context
The colors currently used for the inverted tooltip aren't accessible, causing difficulty for viewers of all eyesights

## Testing
- Tested with storybook

## Screenshots
**Light Mode**
Before | After
-|-
image | image
Notice how the transparency decreases the contrast, making it harder to read |

**Dark Mode**
Before | After
-|-
image | image
Notice how the text is almost completely unreadable |

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by kenchendesign** [comment]
Good touch! I like that you are removing the transparency but I'd suggest using the same dark gray as in the primary text since we avoid absolute black in the UI.

**Comment by alecklandgraf** [comment]
The happo diff is odd 
image

File: packages/core/src/components/Tooltip/styles.ts [hunk 1]
```diff
@@ -55,7 +55,10 @@ export const styleSheetTooltip: StyleSheet = ({ unit, color, pattern, ui }) => (
   },
 
   content_inverted: {
-    backgroundColor: color.accent.blackout,
+    backgroundColor:
+      mode === 'light'
+        ? '#181818' // Dark mode base color
+        : '#FFF', // Light mode base color
```
Comment (alecklandgraf) [inline packages/core/src/components/Tooltip/styles.ts:61]: do these exist on `color`?

**Comment by williaster** [comment]
@alecklandgraf there's a component that falls back to today's date which means that it's happo is non-deterministic. we should fix it, for now I've been approving the diff.

**Comment by Xapphire13** [comment]
@williaster, that's something I also considered. I'll switch to that instead =]

File: packages/core/src/components/Tooltip/styles.ts (hunk 1 above)
Comment (Xapphire13) [reply packages/core/src/components/Tooltip/styles.ts]: Only on the opposite theme 🤔 . Should be solved by using the new `baseInverse`

**Comment by Xapphire13** [comment]
@williaster @alecklandgraf, Your comments should be addressed now ^_^

**Comment by Xapphire13** [comment]
Abandoning in favor of https://github.com/airbnb/lunar/pull/386

**Pull request closed without merging**